src/components/app-top-bar.js

```diff
@@ -294,14 +311,11 @@ const AppTopBar = ({
                 )}
                 onLanguageClick={handleChangeLanguage}
                 language={languageLocal}
-                searchTermDisabled={!isNodeBuilt(currentNode)}
-                initialSearchTerm={
-                    !isNodeBuilt(currentNode)
-                        ? intl.formatMessage({
-                              id: 'InvalidNode',
-                          })
-                        : ''
+                searchTermDisabled={
+                    !isNodeBuilt(currentNode) ||
+                    studyDisplayMode === STUDY_DISPLAY_MODE.TREE
```

Comment:
The test could be : getDisableReason() !== ''   ?

Reply:
:+1: done